apps/docs-next/src/content/reference/extras/use-gltf.mdx

```diff
@@ -116,3 +116,32 @@ Provide types to use your IDEs autocompletion. Use [gltfjsx](https://github.com/
   }
 </script>
 ```
+
+<Tip
+  type="tip"
+  title="How to get the types?"
+>
+  On the [loading-assets](/docs/learn/basics/loading-assets) page, threlte provides the
```

Comment:
Not 100% decided, but I kind of went for a capital T whenever Threlte is mentioned, so "threlte" → "Threlte"